src/src/CustomBuild/define_plugin_sets.h

```diff
@@ -1549,6 +1557,7 @@ To create/register a plugin, you have to :
 #ifdef PLUGIN_SET_COLLECTION_B
     #define USES_P069   // LM75A
 
+    #define USES_P095  // TFT ILI9341, added by TEB 06/28/2024

```

Comment:
When adding plugins to collection sets, you also must make sure this is updated in the documentation.

Reply:
Sorry, should have mentioned that the "customized" build set is a workaround so I can compile with the plugins I need on my workstation. So please ignore the changes in the define_plugin_sets.h

Reply:
Well, I screwed up. I intended to push this to my personal github repo, not the mail ESPEasy repo.

Can you delete this PR?

Instead, review code changes here: https://github.com/thomastech/ESPEasy/tree/Notify_Plugin_Patch

- Thomas